tests/EntityControlTraitTest.php

```diff
@@ -241,9 +244,21 @@ public function testInsertWithoutTimestamps()
         $this->mockInsertDataWithoutTimestamps();
 
         $result = self::$testRepositoryClassWithoutTimestamps->insert([
-            ['name' => 'test_name_1'],
-            ['name' => 'test_name_2'],
-            ['name' => 'test_name_3'],
+            [
+                'name' => 'test_name_1',
+                'created_at' => '1999-01-01',
```

Comment:
let's keep already existed test cases as are and implement separate test case for filling timestamps manually